### PR TITLE
Add matsci.org

### PR DESCRIPTION
**Pull request by wyphan**
Dynamic (`dynamic-theme-fixes.config`) and Filter (`inversion-fixes.config`) modes

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2012,6 +2012,18 @@ INVERT
...
 ================================
 
+matsci.org
+
+INVERT
+.d-header #site-logo
+
+CSS
+.category-logo.aspect-image img {
+  background-color: white !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2022]: Use ${black} and check if that looks still good

File: src/config/inversion-fixes.config [hunk 2]
```diff
@@ -1191,6 +1191,18 @@ img
...
 ================================
 
+matsci.org
+
+NO INVERT
+.d-header #site-logo
+
+CSS
+.category-logo.aspect-image img {
+  background-color: white !important;
```
Comment (Gusted) [inline src/config/inversion-fixes.config:1201]: Same as above

**Comment by wyphan** [comment]
Unfortunately it doesn't work with `${black}`. The main page contains 3 transparent png and 4 white-background png logos. What's the best way to improve readability here? I already tried `CSS background-color: transparent`, but it didn't work for those 4 logos...

Here's the original page with darkreader turned off:
original

Using `${black}` messes up Dynamic mode on Light scheme:
dynamic-light

... as well as Filter mode on Dark theme:
filter-dark

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (wyphan) [reply src/config/dynamic-theme-fixes.config:2022]: Unfortunately not

File: src/config/inversion-fixes.config (hunk 2 above)
Comment (wyphan) [reply src/config/inversion-fixes.config:1201]: Same as above

**Comment by Gusted** [comment]
No suchs thing to get it transparent with CSS :(

**Pull request merged**